lib/find-used-modules.js

```diff
@@ -27,7 +27,7 @@ function parseFile(file, dirname) {
         if (err && addJs) return parseFile(path.join(file, 'index.js')).then(resolve).catch(reject);
         if (err) return reject(err);
         var ast = parse(str.toString(), {
-          ecmaVersion: 9,
+          ecmaVersion: 2023,
```

Comment:
@bilindhajer no particular reason! I see more year-based ECMA versions in typescript so that's probably what I was used to. Happy to choose whichever format you'd prefer @bilindhajer - Looks like `14` is the equivalent for 2023 https://github.com/acornjs/acorn/tree/master/acorn/#interface

Reply:
thanks for the reply!

Reply:
I do share the perspective that we run into year-based version indicators way more often nowadays, so I'm all in favour of switching over.